src/main/kotlin/org/rust/ide/annotator/RsHighlightingAnnotator.kt

```diff
@@ -189,6 +214,6 @@ private fun colorFor(element: RsElement): RsColor? = when (element) {
     is RsTypeAlias -> RsColor.TYPE_ALIAS
     is RsTypeParameter -> RsColor.TYPE_PARAMETER
     is RsConstParameter -> RsColor.CONST_PARAMETER
-    is RsMacroBinding -> RsColor.FUNCTION // TODO FUNCTION?
+    is RsMacroBinding -> RsColor.MACRO
```

Comment:
I'd prefer to have separate highlighting for macro meta variables because otherwise it makes it harder to catch macro meta variables in code.

Reply:
Okay!